test/e2e/requirements.txt

```diff
@@ -3,5 +3,6 @@ requests==2.22.0
 #urllib3==1.26.5
 #jsonschema==2.6.0
 requests_aws4auth
+webdriver-manager
```

Comment:
Unpinned requirements will generate a warning by python `safety`.
```
pip3 install safety
safety check aws-media-insights-content-localization/test/e2e/requirements.txt
```

Reply:
I pinned the packages.  I think Dependabot, which is installed on this project, serves the same purpose as a python `safety` scan, though.